<?xml version="1.0"?>
<launch>
    <node name="osm_bridge" pkg="osm_bridge_ros_wrapper" type="osm_bridge_ros.py" output="screen">
        <param name="overpass_server_ip" value="127.0.0.1" />
        <param name="overpass_server_port" value="8000" />
        <param name="ref_latitude" value="50.1363485" />  <!-- same ref latitude should be used here and for map generation -->
        <param name="ref_longitude" value="8.6474024" /> 
    </node>  
</launch>
